packages/docs/pages/Message/MessagePage.tsx

```diff
@@ -0,0 +1,102 @@
+import { H0, H1, H2, Message, Text } from '@bigcommerce/big-design';
+
+import { Code, CodePreview, Collapsible } from '../../components';
+import { MarginPropTable } from '../../PropTables';
+import { MessagingItemPropTable, MessagingLinkItemPropTable } from '../../PropTables/shared';
+import { MessagePropTable } from '../../PropTables/MessagePropTable';
+
+export default () => (
+  <>
+    <H0>Messages</H0>
+    <Text>An message, used for displaying page/table, feature discovery/system, or non-critical messaging.</Text>
```

Comment:
A message*, however the rest of the copy seems a little weird.

Reply:
I'll give it a rewrite.